apps/o2-blocks/src/callout/constants.js

```diff
@@ -0,0 +1,123 @@
+import colorStudio from '@automattic/color-studio';
+import { __ } from '@wordpress/i18n';
+
+// Extract colors from Color Studio
+const COLORS = colorStudio.colors;
+
+// Configuration constants
+export const ICON_GRID_COLUMNS = 5;
+export const ICON_PICKER_MAX_HEIGHT = '310px';
+export const CALLOUT_TEMPLATE_DEFAULT_COLOR = COLORS[ 'Gray 60' ];
+export const CALLOUT_TEMPLATE_DEFAULT_BACKGROUND_COLOR = COLORS.White;
+
+// Callout template definitions
+export const CALLOUT_TEMPLATES = {
+	custom: {
+		name: __( 'Custom' ),
+		color: CALLOUT_TEMPLATE_DEFAULT_COLOR,
+		icon: 'dashicons-megaphone',
+	},
+	info: {
+		name: __( 'Info' ),
+		color: COLORS[ 'Simplenote Blue 40' ],
+		icon: 'dashicons-info-outline',
+	},
+	tip: {
+		name: __( 'Tip' ),
+		color: COLORS[ 'Celadon 40' ],
+		icon: 'dashicons-lightbulb',
+	},
+	example: {
+		name: __( 'Example' ),
+		color: COLORS[ 'Purple 40' ],
+		icon: 'dashicons-edit-page',
+	},
+	warning: {
+		name: __( 'Warning' ),
+		color: COLORS[ 'Red 40' ],
+		icon: 'dashicons-flag',
+	},
+};
+
+// Color palette for the color picker
+export const CALLOUT_PANEL_COLORS = [
+	{ name: __( 'Simplenote Blue' ), color: COLORS[ 'Simplenote Blue 40' ] },
+	{ name: __( 'Automattic Blue' ), color: COLORS[ 'Automattic Blue 40' ] },
+	{ name: __( 'WordPress Blue' ), color: COLORS[ 'Blue 40' ] },
+	{ name: __( 'WooCommerce Purple' ), color: COLORS[ 'WooCommerce Purple 40' ] },
+	{ name: __( 'Purple' ), color: COLORS[ 'Purple 40' ] },
+	{ name: __( 'Pink' ), color: COLORS[ 'Pink 40' ] },
+	{ name: __( 'Red' ), color: COLORS[ 'Red 40' ] },
+	{ name: __( 'Orange' ), color: COLORS[ 'Orange 40' ] },
+	{ name: __( 'Yellow' ), color: COLORS[ 'Yellow 40' ] },
+	{ name: __( 'Green' ), color: COLORS[ 'Green 40' ] },
+	{ name: __( 'Jetpack Green' ), color: COLORS[ 'Jetpack Green 40' ] },
+	{ name: __( 'Celadon' ), color: COLORS[ 'Celadon 40' ] },
+	{ name: __( 'Gray 40' ), color: COLORS[ 'Gray 40' ] },
+	{ name: __( 'Gray 60' ), color: COLORS[ 'Gray 60' ] },
+	{ name: __( 'Gray 80' ), color: COLORS[ 'Gray 80' ] },
```

Comment:
what is the shape of the data in the colors package? does it not have the name of the colors? it seems like we’re recreating a lot of the data from that, though I’m definitely not familiar with it so I don’t know. is it missing color names that it should have?